docs/configuration/index.md

```diff
@@ -616,9 +616,10 @@ the [HDFS input source](../ingestion/input-sources.md#hdfs-input-source).
 You can set the following property to specify permissible protocols for
 the [HTTP input source](../ingestion/input-sources.md#http-input-source).
 
-|Property|Possible values|Description|Default|
-|--------|---------------|-----------|-------|
-|`druid.ingestion.http.allowedProtocols`|List of protocols|Allowed protocols for the HTTP input source.|`["http", "https"]`|
+|Property| Possible values | Description                                                                                                                               |Default|
+|--------|-----------------|-------------------------------------------------------------------------------------------------------------------------------------------|-------|
+|`druid.ingestion.http.allowedProtocols`| List of protocols | Allowed protocols for the HTTP input source.                                                                                              |`["http", "https"]`|
+|`druid.ingestion.http.allowedHeaders`| A list of permitted request headers for the HTTP input source. By default, the list is empty, which means all headers are allowed in the ingestion specification. |`[]`|
```

Comment:
when the list is empty, no header is allowed, correct?

Reply:
default value for druid.ingestion.http.allowedHeaders is empty list which means all headers are allowed.

Reply:
if we we want to restrict headers then druid.ingestion.http.allowedHeaders should be non empty.

Reply:
made changes to modify behavior. Default value for druid.ingestion.http.allowedHeaders is empty which allows no headers to pass.